website/templates/roomview.html

```diff
@@ -2,39 +2,64 @@
 {% block page_content %}
     <h1> Kitchen </h1>
     <br>
+
     <ul class="nav nav-pills">
-        <li role="presentation" class="active"><a href="#">Manage Devices</a></li>
+       <li role="presentation" class="dropdown active">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="True"
+               aria-expanded="False">
+                <span class="glyphicon glyphicon glyphicon-plus" aria-hidden="true"></span> Link Device to Room
+            </a>
+            <ul class="dropdown-menu">
+                <div class="input-group">
+                    <input type="text" class="form-control" placeholder="Room Name" aria-describedby="basic-addon2">
+                    <span class="input-group-addon" id="basic-addon2"> <span
+                            class="glyphicon glyphicon-plus"></span></span>
+                </div>
+            </ul>
+       </li>
+
+        <li role="presentation" class="dropdown active">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="True"
+               aria-expanded="False">
+                <span class="glyphicon glyphicon-minus" aria-hidden="true"></span> Unlink Device
+            </a>
+            <ul class="dropdown-menu">
+                <div class="input-group">
+                    <input type="text" class="form-control" placeholder="Room Name" aria-describedby="basic-addon2">
+                    <span class="input-group-addon" id="basic-addon2"> <span
+                            class="glyphicon glyphicon-minus"></span></span>
+                </div>
+            </ul>
+        </li>
+      <li role="presentation"><a href="{{ url_for('.devices') }}">Manage Devices</a></li>
     </ul>
+
+
     <br>
     <table class="table table-inverse">
```

Comment:
Have you tried using the Bootstap responsive grid system instead of one table for the different kinds of devices? I think it might give us more flexibility as to the different information that we can show depending on the type.